src/useMutate.test.tsx

```diff
@@ -447,6 +465,34 @@ describe("useMutate", () => {
       expect(mutate0).toEqual(mutate1);
       expect(mutate0).toEqual(mutate2);
     });
+
+    it("should remain the same across calls with deeply equal props", async () => {
+      nock("https://my-awesome-api.fake")
+        .post("/plop/one")
+        .reply(200, { id: 1 })
+        .persist();
+
+      const wrapper: React.FC = ({ children }) => (
+        <RestfulProvider base="https://my-awesome-api.fake">{children}</RestfulProvider>
+      );
+      const getPath = ({ id }: { id: string }) => `plop/${id}`;
+      const { result } = renderHook(
+        () =>
+          useMutate<{ id: number }, unknown, {}, {}, { id: string }>("POST", getPath, {
+            pathParams: { id: "one" },
+          }),
+        {
+          wrapper,
+        },
+      );
+      const mutate0 = result.current.mutate;
+      const mutate1 = result.current.mutate;
+      await result.current.mutate({});
+      const mutate2 = result.current.mutate;
+
+      expect(mutate0).toEqual(mutate1);
+      expect(mutate0).toEqual(mutate2);
```

Comment:
Is it working with `.toBe` instead of `.toEqual`? (`.toEqual` don't insure that the ref is the same https://jestjs.io/docs/expect#toequalvalue)

Reply:
Does indeed! Pushed that fix.